Microsoft.Alm.Authentication/Microsoft.Alm.Authentication.csproj

```diff
@@ -59,13 +59,15 @@
       <HintPath>..\packages\Microsoft.IdentityModel.Clients.ActiveDirectory.2.18.206251556\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="PresentationFramework" />
```

Comment:
The `Microsoft.Alm.Authentication` library cannot have a dependency on a UX framework. It is used, internally, at Microsoft for a number of VSTS authentication scenarios; most of which are services.